docs/user_guide/bank/pm_partner_profiles.md

```diff
@@ -36,7 +52,11 @@ The high level sections of the partner profile are:
 - Additional Documents
 - Settings (not configurable)
 
+Unless marked otherwise, above,  you can choose whether you and the partners will see these sections,  in your "My Organization" section (see ["Getting started - customization](getting_started_customization.md))
+
+[!NOTE] The section screenshot images below have sample data in them.  Your initial screens may look a bit different as a result.
```

Comment:
Github apparently wants the `NOTE` call-out (as well as the other call-out types) to have a newline between label and text and to be block quoted:

```suggestion
> [!NOTE]
> The section screenshot images below have sample data in them.  Your initial screens may look a bit different as a result.
```